test/fulfillment_order_test.rb

```diff
@@ -227,6 +227,57 @@ def setup
       end
     end
 
+    context "#open" do
+      should "be able to open fulfillment order" do
+        fulfillment_order = ShopifyAPI::FulfillmentOrder.find(519788021)
+        fulfillment_order.status = 'scheduled'
+
+        opened = ActiveSupport::JSON.decode(load_fixture('fulfillment_order'))
+        opened['status'] = 'open'
```

Comment:
Why do we set the status here before making the `POST`?

Reply:
Oh, we're comparing it against the fixture, I see.

Reply:
This fixture has the status `open` already actually, here I am trying to make sure it is `open`, it should be fine without it. Just clearer this way IMO.